elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -530,8 +536,16 @@ export class PeerPool extends EventEmitter {
 		return peers;
 	}
 
-	public getUniqueOutboundConnectedPeers(): ReadonlyArray<P2PPeerInfo> {
-		return getUniquePeersbyIp(this.getAllConnectedPeerInfos(OutboundPeer));
+	public getIsConnectedPeerByIP(
+		peerInfo: P2PPeerInfo,
+		kind: typeof OutboundPeer | typeof InboundPeer,
+	): boolean {
+		const connectedPeers = this.getAllConnectedPeerInfos(kind);
+
+		return connectedPeers.filter(peer => peer.ipAddress === peerInfo.ipAddress)
+			.length > 0
+			? true
```

Comment:
the expression `length > 0` already returns a boolean value, so you can remove the conditional ternary operator.